src/Microsoft.DotNet.Cli.Utils/Microsoft.DotNet.Cli.Utils.xproj

```diff
@@ -10,7 +10,7 @@
     <ProjectGuid>61b7c351-c77d-43f7-b56f-bb1440178e10</ProjectGuid>
     <RootNamespace>Microsoft.DotNet.Cli.Utils</RootNamespace>
     <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)'=='' ">..\..\artifacts\obj\$(MSBuildProjectName)</BaseIntermediateOutputPath>
-    <OutputPath Condition="'$(OutputPath)'=='' ">..\..\artifacts\bin\$(MSBuildProjectName)\</OutputPath>
+    <OutputPath Condition="'$(OutputPath)'=='' ">..\..\artifacts\bin</OutputPath>
```

Comment:
I would vote for **not** forking the output directories between the build from command line and VS projects. It makes it so much nicer when there is only a single output directory, no matter where you are.